.github/workflows/ci.yml

```diff
@@ -89,7 +86,7 @@ jobs:
       - uses: ./.github/actions/setup-node-env
 
       - name: Run Storybook smoke-test
-        run: pnpm storybook -- --ci --smoke-test
+        run: pnpm storybook:smoke
```

Comment:
Another thing occurring to me, sort of as a consequence of our discussion about the playwright version script.

I wonder if things that aren't used (or meant to be used) generally, like this smoke test script, should actually be in the package.json at all. It makes the list of scripts longer just to slightly shorten the already not-too-complex one-liner in the CI scripts.

Reply:
That does make sense, a lot of scripts could be eliminated if we only keep those in the `package.json` file that are used frequently and/or mostly invoked manually.